powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/CallbackUrlEntity.java

```diff
@@ -90,6 +92,25 @@ public class CallbackUrlEntity implements Serializable {
     @Convert(converter = CallbackAuthenticationConverter.class)
     private CallbackUrlAuthenticationEntity authentication;
 
+    /**
+     * Maximum number of attempts to send the callback.
+     */
+    @Column(name = "max_attempts")
+    private Integer maxAttempts;
```

Comment:
We added new columns, do we handle correctly the migration of rows which already exist? I assume null values would be used, is this OK.

Reply:
Yes, null values are fine. For that reason there are configurable defaults on application properties level.